crates/nu-parser/src/parser.rs

```diff
@@ -4553,6 +4553,60 @@ pub fn parse_operator(
                 )),
             );
         }
+        b"&" => {
+            return (
+                garbage(span),
+                Some(ParseError::UnknownOperator(
+                    "&",
+                    "Did you mean 'bit-and'?",
+                    span,
+                )),
+            );
+        }
+        b"<<" => {
+            return (
+                garbage(span),
+                Some(ParseError::UnknownOperator(
+                    "<<",
+                    "Did you mean 'bit-shl'?",
+                    span,
+                )),
+            );
+        }
+        b">>" => {
+            return (
+                garbage(span),
+                Some(ParseError::UnknownOperator(
+                    ">>",
+                    "Did you mean 'bit-shr'?",
```

Comment:
we might also want to point people who want to use the bash append `>>` to how to append in nu

Reply:
Yeah good point. But that would probably require a similar catching mechanism like in your PR #7241 (Might be a good fit there to avoid merge conflicts)